src/sentry/api/endpoints/organization_onboarding_tasks.py

```diff
@@ -7,15 +7,30 @@
     OrganizationEndpoint, OrganizationPermission
 )
 
-from sentry.models import OnboardingTaskStatus, OrganizationOnboardingTask
+from sentry.models import OnboardingTask, OnboardingTaskStatus, OrganizationOnboardingTask
+from sentry.receivers import check_for_onboarding_complete
 
 
 class OrganizationOnboardingTaskEndpoint(OrganizationEndpoint):
     permission_classes = [OrganizationPermission, ]
 
     def post(self, request, organization):
-        if request.DATA['status'] == 'skipped':
-            OrganizationOnboardingTask.objects.create_or_update(
+        try:
+            task_id = int(request.DATA['task'])
+        except ValueError:
```

Comment:
`int()` could also raise a `TypeError`, so this needs to be `except (TypeError, ValueError)`